README.md

```diff
@@ -0,0 +1,225 @@
+
+NOTE: At some point in the future we may integrate automatically into OperatorHub so that OSP Director Operator is available automatically in your OCP installations default OLM Catalog sources.
+
+## Deploying OpenStack once you have the OSP Director Operator installed
+
+1) Define your OpenStackNet custom resource. At least 1 network is required for the ctlplane. Optionally you may define multiple networks to be used with TripleO's network isolation architecture. In addition to IP address assignment the OpenStackNet includes information that is used to define the network configuration policy used to attach any VM's to this network via OpenShift Virtualization. The following is an example of a simple IPv4 ctlplane network which uses linux bridge for its host configuration.
+```yaml
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackNet
+metadata:
+  name: ctlplane
+spec:
+  cidr: 192.168.25.0/24
+  allocationStart: 192.168.25.100
+  allocationEnd: 192.168.25.250
+  gateway: 192.168.25.1
+  attachConfiguration:
+    bridgeName: br-osp
+    nodeNetworkConfigurationPolicy:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      desiredState:
+        interfaces:
+        - bridge:
+            options:
+              stp:
+                enabled: false
+            port:
+            - name: enp6s0
+          description: Linux bridge with enp6s0 as a port
+          name: br-osp
+          state: up
+          type: linux-bridge
+```
+
+If you write the above YAML into a file called ctlplane-network.yaml you can create the OpenStackNet via this command:
+
+```bash
+oc create -f ctlplane-network.yaml
+```
+
+2)Create a [ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) which define any custom Heat environments and Heat templates used for TripleO network configuration. Any adminstrator defined Heat environment files can be provided in the ConfigMap and will be used as a convention in later steps used to create the Heat stack for Overcloud deployment. As a convention each OSP Director Installation will use 2 ConfigMaps named 'tripleo-deploy-config-custom' and 'tripleo-net-config' to provide this information.
+
+A good example of ConfigMaps that can be used can be found in our [dev-tools](https://github.com/openstack-k8s-operators/osp-director-dev-tools) GitHub project. Direct links for each the sample files can be found below:
+
+-[Net-Config files](https://github.com/openstack-k8s-operators/osp-director-dev-tools/tree/master/ansible/files/osp/net_config)
+
+-[Tripleo Deploy custom files](https://github.com/openstack-k8s-operators/osp-director-dev-tools/tree/master/ansible/templates/osp/tripleo_deploy) (NOTE: these are Ansible templates and need to have variables replaced to be used directly!)
```

Comment:
and https://github.com/openstack-k8s-operators/osp-director-dev-tools/tree/master/ansible/files/osp/tripleo_deploy for the network-environment.yaml . maybe we should just move it to the templates

Reply:
It might even be better if we have static examples for some of these within the osp-director-operator itself. Linking to these is fine for starters but eventually once things stabilize linking across repos seems a bit dirty.

Reply:
I was going to take this as a follow up to keep it less verbose here in our initial README btw